lib/ae_page_objects/core/dsl.rb

```diff
@@ -20,7 +20,9 @@ def element(name, options = {}, &block)
 
       self.element_attributes[name.to_sym] = klass
 
-      define_method name do |&block|
+      define_method name do |&block_for_define_method|
```

Comment:
Yes, if the developer write that code the block is never called just like any other Ruby method.